include/boost/parameter/python.hpp

```diff
@@ -64,12 +63,24 @@ namespace boost { namespace parameter { namespace python { namespace aux
           Py_TPFLAGS_DEFAULT,               /* tp_flags       */
           0,                                /* tp_doc         */
       };
+      
+#if PY_MAJOR_VERSION <= 2
```

Comment:
I think that the `Py_TYPE(o)` macro is supposed to encapsulate the differences between 2 and 3, so if you use it, there would be no need to check the version.